tiled/client/utils.py

```diff
@@ -94,6 +95,24 @@ def __init__(self, message, request, response):
         super().__init__(message=message, request=request, response=response)
 
 
+def should_retry(exc: Exception) -> bool:
+    # If the error is an HTTP status error, only retry on 5xx errors.
+    if isinstance(exc, httpx.HTTPStatusError):
+        return exc.response.status_code >= 500
+
+    # Otherwise retry on all httpx errors.
+    return isinstance(exc, httpx.HTTPError)
```

Comment:
Tired eyes read this as `exec` for execute. 
```suggestion
def should_retry(exception: Exception) -> bool:
    # If the error is an HTTP status error, only retry on 5xx errors.
    if isinstance(exception, httpx.HTTPStatusError):
        return exception.response.status_code >= 500

    # Otherwise retry on all httpx errors.
    return isinstance(exception, httpx.HTTPError)
```